### PR TITLE
Setup of mxd tutorial

### PR DESCRIPTION
**Pull request by jkbquabeck**
## Description

This PR publishes the setup section of the reworked MXD tutorial. This was requested in issue #315 (https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/issues/315). The MXD tutorials are to be published in different repositories, starting with tutorial_resources.

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [ ] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Comment by hemantxpatel** [comment]
@jkbquabeck Can you make it a regular PR so that it could be reviewed and merged.

File: mxd/2_MXD_Setup.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
+# Basic dataspace setup
+
+The "MXD" dataspace initially consists of several components: `Alice` and `Bob` (two Tractus-X EDC connectors),
+a `vault` instance each, a Postgres database, a Managed Identity Wallet app, a Keycloak instance. `Alice` and `Bob` will
```
Comment (paullatzelsperger) [inline mxd/2_MXD_Setup.md:4]:
```suggestion
a Vault instance each, a Postgres database, a Managed Identity Wallet app, a Keycloak instance. `Alice` and `Bob` will
```

File: mxd/2_MXD_Setup.md [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+be our dataspace participants. Each of them stores their secrets in their "private" vault instance, and there is a
```
Comment (paullatzelsperger) [inline mxd/2_MXD_Setup.md:5]:
```suggestion
be our dataspace participants. Each of them stores their secrets in their respective vault instances, and there is a
```

File: mxd/2_MXD_Setup.md [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+shared Postgres server, where each of them has a database. MIW and Keycloak are "central" components, they only exist
```
Comment (paullatzelsperger) [inline mxd/2_MXD_Setup.md:6]:
```suggestion
shared Postgres server, where each of them has a database. MIW and Keycloak are central components, they only exist
```

File: mxd/2_MXD_Setup.md [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+
+Wait. Then wait some more. It will take a couple of minutes until all services are booted up. If your machine is a
+potato, it'll take even longer. Just get a coffee. Eventually, it should look similar to this:
+
+![img.png](assets/img.png)
+
+## Inspect terraform output
+
+After the `terraform` command has successfully completed, it will output a few configuration and setup values
+that we will need in later steps. Please note that most values will be different on your local system.
```
Comment (paullatzelsperger) [inline mxd/2_MXD_Setup.md:39]:
```suggestion
that we will need in later steps. Please note that some values will be different on your local system.
```

**Review by paullatzelsperger — COMMENTED** [review]
some minor nits inline

**Review by paullatzelsperger — APPROVED** [review]

**Pull request merged**